src/analytics/tangle/milestone_size.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use super::BlockAnalytics;
+use crate::types::{
+    ledger::BlockMetadata,
+    stardust::block::{Block, Payload},
+    tangle::MilestoneIndex,
+};
+
+/// Milestone size statistics.
+#[derive(Clone, Debug, Default)]
+pub struct MilestoneSizeMeasurement {
+    total_milestone_payload_bytes: u64,
+    total_tagged_data_payload_bytes: u64,
+    total_transaction_payload_bytes: u64,
+    total_treasury_transaction_payload_bytes: u64,
+    total_milestone_bytes: u64,
```

Comment:
Maybe we can use `usize`s here? If InfluxDB can't handle `usize` our writer should take care of the conversion.

Reply:
fair enough ... I'll change it.